docs/kqueen.rst

```diff
@@ -153,6 +153,110 @@ in the configuration file, set the environment variable matching the KQUEEN_<con
       - Addresses allowed to access metrics endpoint without token
 
 
+External services deployment
+-------
+1. **Members managing** - To provide user registration by invitations from other users by emails, mail server should be set up. 
+It is possible to use kqueen predefined mail service or to run new one. Example configuration mail service in docker-compose.production.yml:
+
+.. code-block:: yaml
+   mail:
+     image: modularitycontainers/postfix
+     volumes:
+       - /var/spool/postfix:/var/spool/postfix
+       - /var/mail:/var/spool/mail
+     environment:
+       MYHOSTNAME: 'mail'
+
+Configure following variables in kqueen ui service:
+
+.. code-block:: yaml
+   KQUEENUI_MAIL_SERVER: mail
+   KQUEENUI_MAIL_PORT: 10025
+
+Pay attention that volume-mapping for mail containers is additional feature. It used for store mailing history and forward additional postfix mail configuration (it should configured properly on local machine),
+otherwise its possible to run mail server without volume-mapping. After all configuration steps, user should invite members using email notification. Member should receive mail with activation link to Kqueen service and possibility of password-setting.
+Superadmin also can manage member roles.
+
+2. **Metrics collecting** - Its possible to add external Prometheus serveri to extend monitoring in Kqueen. In case of using external server, need to include rules from kqueen/prod/prometheus into existing Prometheus service. Otherwise, its possible to use predefined Prometheus, defined in docker-compose.production.yml.
+Configure prometheus service (ip address/port, volumes) in docker-compose.production.yml. Define  scraper IP address for PROMETHEUS_WHITELIST variable in kqueen api service section.
+All application metrics exported on **/metrics** and this metric can scraped by any external Prometheus instance. There two ways to authenticate to get this endpoint:
+
+- Kqueen API
+.. code-block:: bash
+   TOKEN=$(curl -s -H "Content-Type: application/json" --data '{"username":"admin","password":"default"}' -X POST <<kqueen_api_host>>:5000/api/v1/auth | jq -r '.access_token'); echo $TOKEN
+   curl -H "Authorization: Bearer $TOKEN" <<kqueen_api_host>>:5000/metrics/
+
+
+- Prometheus API
+Add scraper IP address to PROMETHEUS_WHITELIST configuration and then:
+
+.. code-block:: bash
+   curl <<prometheus_host>>:<<prometheus_port>>/metrics
+
+
+Provisioner configuration settings
```

Comment:
Provision a Kubernetes cluster?

Please add an intro, for example:

You can provision a Kubernetes cluster using various community installers or provisioners, such as ...

**Provision a Kubernetes cluster using the Google Kubernetes Engine:**
1.
2.
3.

**Provision a Kubernetes cluster using the Azure Kubernetes Service:**
1.
2.
3.

...